docs/UserGuide.adoc

```diff
@@ -18,13 +18,23 @@ By: `Team T12-3`      Since: `Feb 2019`      Licence: `MIT`
 
 == Introduction
 
-MediTabs is for those who prefer to use a desktop app for managing medicine stock taking. This is especially useful for clinics such as family clinics or private clinics to track the amount of medicine they have left available. More importantly, MediTabs is optimized for those who prefer to work with a Command Line Interface (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, MediTabs can get your medicine stock taking done faster than traditional GUI apps. Interested? Jump to the <<Quick Start>> to get started. Enjoy!
+Welcome to the _MediTabs User Guide_. You can find step by step instructions on how to set up MediTabs and take advantage of the features provided by MediTabs to optimise your clinic's medicine stock taking workflow. Interested? You can jump to the <<Quick Start>> to start get started. Enjoy!
+
+=== What is MediTabs?
+
+MediTabs is a desktop application for managing medicine stock taking. You can use MediTabs in your clinics such as family clinics or private clinics to keep track of your clinic's medicine inventory such as which medicines are low in stock and which medicines are expiring soon.
+
```

Comment:
The second sentence is a bit grammatically wrong. Maybe can change to " You can use MediTabs in your clinics to easily track of your medicine inventory and take note of important information such as which medicines are low in stock and which medicines are expiring soon."

Reply:
Good suggestion. Thank you for pointing it out I will resolve it in future commits most likely tomorrow